orchestrator-salt/src/main/resources/salt-common/salt/filecollector/template/cdp-doctor-commands.yaml.j2

```diff
@@ -6,6 +6,14 @@ commands:
       report: false
     - command: netstat -tlpn
       output: /tmp/doctor_netstat.txt
+    - command: free -h
+      output: /tmp/doctor_free.txt
+    - command: ps -eo pid,ppid,cmd,%mem,%cpu --sort=-%mem | head
+      output: /tmp/doctor_top_proc.txt
+    - command: df -h
+      output: /tmp/doctor_disk_full.txt
```

Comment:
why is the name dist_full.txt? df is short for disk free

Reply:
it's just accidental...so i have created one check for only the **logs** and one for the **full** filesystem based on the command name. can be changed if you think disk free as name is better (to be alias of the command)

Reply:
whatever, fixed it to not confuse anyone by the naming